HW02/Alexey_Zalyotov/alexey_zalyotov_hw_02_t_03.rb

```diff
@@ -0,0 +1,34 @@
+require 'date'
+
+def events_timestamps
+  # logs information from file to array
+  logs = File.readlines("#{__dir__}/fixtures/data_hw_02_t_03.txt", chomp: true)
+  # find lines with "Calling core with action" substr and parse their timestamps
+  logs.select { |line| line =~ /.*Calling core.*/ }.map do |substr|
+    substr[/\S+\s\S+/]
+  end
+end
+
+if events_timestamps.size >= 2
+  # convert timestamps into date_time format
+  datetimestamps = events_timestamps.map do |str|
+    DateTime.strptime(str[/\S+\s\S+/], '%Y-%m-%d %H:%M:%S.%L')
```

Comment:
Maybe just `DateTime.parse`? What do you think?
```suggestion
    DateTime.parse(str[/\S+\s\S+/])
```

Reply:
Done